include/cpu.h

```diff
@@ -522,6 +522,9 @@ struct CPUBlock {
 
 extern CPUBlock cpu;
 
+void CPU_Init(Section* sec);
+void CPU_ShutDown(Section* sec);
```

Comment:
IMHO either `MODULE_ShutDown`, or `MODULE_Destroy` - not both.

Reply:
Yeah, I don't like Destroy... Should be Init and Shutdown, IMO; those are standard terms. And the word is "shutdown", so no weird ShutDown capitalisation.

Reply:
Yeah I've left the function names as they were but they should be standardized across the codebase.